bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/SortedLedgerStorage.java

```diff
@@ -78,6 +79,7 @@ public void initialize(ServerConfiguration conf,
                 .setNameFormat("SortedLedgerStorage-%d")
                 .setPriority((Thread.NORM_PRIORITY + Thread.MAX_PRIORITY) / 2).build());
         this.stateManager = stateManager;
+        this.isTransactionalCompactionEnabled = conf.getUseTransactionalCompaction();
```

Comment:
I don't think we need this flag any more, no?

Reply:
yes, it is not needed. Will remove.